.github/workflows/test.yml

```diff
@@ -392,7 +392,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v1
         with:
-          python-version: 3.6
+          python-version: 3.7
```

Comment:
Does this line means we are using Python3.7 as our default environment for all the tests?

Reply:
No that's just for a single test (Test backend - use azure as default bundle store) that was previously using python3.6. I could also change that test to use Ubuntu20.04 instead, but I didn't know why only that test was using Python3.6 so I changed it to Python3.7